runtime/src/anchor/mod.rs

```diff
@@ -36,9 +36,6 @@ const MAX_LOOP_IN_TX: u64 = 500;
 /// date 3000-01-01 -> 376200 days from unix epoch
 const STORAGE_MAX_DAYS: u32 = 376200;
 
-/// The child info for this module
-const CHILD_INFO: ChildInfo<'static> = ChildInfo::new_default(b"anchor");
```

Comment:
this is the breaking change

Reply:
yeap

Reply:
There was another benefit of the prefix though IIRC – it implied that you could delete all keys with one command (by prefix). This would no longer work, right?

Reply:
that would no longer work with the current implementation. We could still have a prefix but that would involve a migration. Since we are anyway doing a migration, my hope is we could add this prefix then

Reply:
How will we do the migration? Shouldn't the migration be part of this upgrade?

Reply:
That part is bit tricky and migrations are not part of this PR. this PR ensures we have the node upgrade done. We would still need to figure out migrations for runtime. I'm currently talking to parity regarding this specific migration